app/src/main/java/com/example/anuj/imdbmovielist/MovieDetail.java

```diff
@@ -0,0 +1,128 @@
+package com.example.anuj.imdbmovielist;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.drawable.GlideDrawable;
+import com.bumptech.glide.request.RequestListener;
+import com.bumptech.glide.request.target.Target;
+
+import java.util.ArrayList;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class MovieDetail extends AppCompatActivity {
+    TextView title, type, year, imdbid, imageUri;
```

Comment:
imdbid -> imdbId
title, type, year, etc. can denote a string or other values as well. A better naming convention could be something like titleView, or titleTextView